mylyn.docs/wikitext/core/org.eclipse.mylyn.wikitext.maven/pom.xml

```diff
@@ -42,12 +42,12 @@
 		<dependency>
 			<groupId>org.eclipse.mylyn.docs</groupId>
 			<artifactId>org.eclipse.mylyn.wikitext</artifactId>
-			<version>${project.version}</version>
+			<version>3.0.48.202308291007</version>
 		</dependency>
 		<dependency>
 			<groupId>org.eclipse.mylyn.docs</groupId>
 			<artifactId>org.eclipse.mylyn.wikitext.textile</artifactId>
-			<version>${project.version}</version>
+			<version>3.0.48.202308291007</version>
```

Comment:
Why did we here use an old Version?
In https://download.eclipse.org/mylyn/updates/release/ we already have Version 4.2.0.v20240302-0923

Reply:
I mentioned this in the PR comment, because we don't have the new one published